File: common_library/src/main/java/com/phone/common_library/custom_view/ColorUiInterface.java
package com.phone.common_library.custom_view;

import android.content.res.Resources;
import android.view.View;

/**
 * 换肤接口
 * Created by chengli on 15/6/8.
 */
public interface ColorUiInterface {

    View getView();

    void setTheme(Resources.Theme themeId);
}

File: library_common/src/main/java/com/phone/common_library/custom_view/ColorUiInterface.kt
package com.phone.common_library.custom_view

import android.content.res.Resources.Theme
import android.view.View

/**
 * 换肤接口
 * Created by chengli on 15/6/8.
 */
interface ColorUiInterface {

    fun getView(): View?

    fun setTheme(themeId: Theme?)
}